keystores/src/test/java/com/palantir/remoting3/config/ssl/SslConfigurationTest.java

```diff
@@ -28,18 +28,18 @@
     @Test
     public void testSerDe() throws Exception {
         SslConfiguration serialized = SslConfiguration.builder()
-                .trustStorePath(Paths.get("truststore.jks"))
+                .trustStorePath(Paths.get("/truststore.jks"))
                 .trustStoreType(SslConfiguration.StoreType.JKS)
-                .keyStorePath(Paths.get("keystore.jks"))
+                .keyStorePath(Paths.get("/keystore.jks"))
                 .keyStoreType(SslConfiguration.StoreType.JKS)
                 .keyStoreKeyAlias("alias")
                 .keyStorePassword("password")
                 .build();
-        String deserializedCamelCase = "{\"trustStorePath\":\"truststore.jks\",\"trustStoreType\":\"JKS\","
-                + "\"keyStorePath\":\"keystore.jks\",\"keyStorePassword\":\"password\","
+        String deserializedCamelCase = "{\"trustStorePath\":\"file:///truststore.jks\",\"trustStoreType\":\"JKS\","
```

Comment:
can old jackson versions deserialize this? (asking because some services (SMM?) may use the SslConfiguration class over the wire, I believe)

Reply:
I think I went overboard here with making behaviour the same in both cases. Let me revert that

Reply:
Added extra cases to show the compatibility guarantees